client/landing/stepper/declarative-flow/newsletter.ts

```diff
@@ -118,6 +124,16 @@ const newsletter: Flow = {
 						);
 					}
 
+					// @TODO: REMOVE BEFORE MERGE
+					/* eslint-disable no-console */
+					console.log( 'newsletter.ts flow controller:', providedDependencies?.siteSlug, goals );
+					/* eslint-enable */
+
+					// Save an intention to set up paid subscribers as a "goal"
+					if ( providedDependencies?.paidSubscribers && providedDependencies?.siteSlug ) {
+						setGoalsOnSite( providedDependencies.siteSlug, goals );
```

Comment:
Feels like this should be at `newsletter-setup` step

Reply:
but there's no site at that point right?